json-endpoints/class.wpcom-json-api-list-posts-v1-2-endpoint.php

```diff
@@ -375,7 +375,10 @@ function callback( $path = '', $blog_id = 0 ) {
 						if ( ! isset( $return[$key] ) ) {
 							$return[$key] = (object) array();
 						}
-						$return[$key]->next_page = $this->build_page_handle( $last_post, $query );
+
+							if ( isset( $last_post['ID'] ) ) {
+								$return[ $key ]->next_page = $this->build_page_handle( $last_post, $query );
+							}
```

Comment:
```suggestion
						if ( isset( $last_post['ID'] ) ) {
							$return[ $key ]->next_page = $this->build_page_handle( $last_post, $query );
						}
```

Reply:
php-changed flagged those lines as needed the extra tab. I didn't run phpcs on the whole file, but I can do that.

Reply:
92e5123